### ISSUE TITLE
Exclude a range of folios from a manifest

### ISSUE THREAD

**Issue by lguariento** [DDMAL/diva.js#547]
Hello,

is is possible to tell diva.js to ignore a range of canvases from a IIIF manifest.json? Say, I want to avoid displaying from canvas 2 to canvas 21. I've edited the manifest removing those canvases. That messes up with the GoTo function though (the top-right built-in 'Go').

**Comment by ahankinson** [comment]
I don't think this is possible; Diva generates an index of the manifest on first load.

That said, you can fetch the manifest outside of Diva, manipulate it as JSON, and then pass the manipulated object directly to Diva to load, rather than trying to edit it directly in Diva.
